### PR TITLE
Skip configuration parameter for skipping execution fixes #161

### LINKED ISSUE #161: Skip configuration parameter for skipping execution
Discussed here : http://discuss.asciidoctor.org/Maven-plugin-skip-td3135.html

### PR DESCRIPTION
**Pull request by vietj**

### CODE REVIEW

**Comment by abelsromero** [comment]
Simple and functional, I like it :+1: Just one small note about the README, can you add the detail on the default value and change the capital letter?
I know I'm being picky, but I would like to keep all descriptions with the same format.

**Comment by vietj** [comment]
sure

File: src/main/java/org/asciidoctor/maven/AsciidoctorMojo.java [hunk 1]
```diff
@@ -135,6 +138,10 @@
 
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
+        if (skip) {
+            getLog().info( "Asciidoctor generation is skipped." );
```
Comment (mojavelinux) [inline src/main/java/org/asciidoctor/maven/AsciidoctorMojo.java]: Could you change this message to:

> AsciiDoc processing is skipped.

**Comment by mojavelinux** [comment]
Thanks @vietj! I added one inline comment. Other than that, this looks great.

**Comment by vietj** [comment]
no problem, will do all of that today.

**Comment by vietj** [comment]
done for both.

**Comment by mojavelinux** [comment]
:+1:

**Comment by vietj** [comment]
btw, is there a release soon of this maven plugin ?

**Comment by abelsromero** [comment]
> btw, is there a release soon of this maven plugin ?

That's really a good question. The thing is relases are aligned with asciidoctor and asciidoctorj. Once asciidoctor releases, asciidoctorj tests and releases, and then the maven/gradle plugin (and many others i guess).
As for the maven plugin and asciidoctorj there are some nice improvements, just by chance, on extension support for both projects. So, I feel that a relase makes sense, as for asciidoctor itself I cannot talk.
Anyway, maybe we could discuss this in the forums, wdyt?

**Comment by mojavelinux** [comment]
I'm working on the 1.5.3 release right now. I'm trying to get it out in early June. Once that happens, there will be a train of releases :)

**Comment by vietj** [comment]
good to know, it's not blocker for Vert.x but if we have it we can remove maven workaround and the associated bloat :-)

**Comment by mojavelinux** [comment]
I'm still okay with a micro release of the plugin though. It would be named 1.5.2.1.

**Comment by vietj** [comment]
Vert.x 3 is for mid june, so if you release early June, it should be fine

**Comment by vietj** [comment]
next week we have the last milestone

**Pull request merged**